sentry-android-fragment/src/main/java/io/sentry/android/fragment/FragmentLifecycleIntegration.kt

```diff
@@ -20,7 +20,7 @@ class FragmentLifecycleIntegration(
     Integration,
     Closeable {
 
-    constructor(application: Application) : this(application, true, false)
+    constructor(application: Application) : this(application, true, true)
```

Comment:
Can we rather flip this flag during the `FragmentLifecycleIntegration` creation? otherwise, it'd require a major bump I believe, cc @bruno-garcia

Reply:
Yeah, sounds like a better way for sure 👍

Reply:
Good point Manoel!